docs/index.rst

```diff
@@ -1,14 +1,125 @@
-Unified Workflow
-================
-
-TBD
+.. include:: sections/links.rst
 
 .. toctree::
    :hidden:
 
    sections/user_guide/index
    sections/contributor_guide/index
 
+
+Unified Workflow
+================
+
+``uwtools`` is an open-source modern Python package that helps automate common tasks needed for many standard numerical weather prediction (NWP) workflows and wraps Unified Forecast System (UFS) components in a highly configurable run-time interface (a driver) that maximizes flexibility, interoperability, and usability for any UFS Application.
+
+If you're interested in contributing, check out the :doc:`Contributor Guide <sections/contributor_guide/index>`.
+
+For users who want to start using the toolbox and framework, take a peek at our :doc:`User Guide <sections/user_guide/index>`.
+
+.. contents:: On this page
+   :depth: 2
+   :local:
+
+
+
+The Tools
+---------
+
+All our tools are available by command line interface (CLI), with select operations supported in the Python API. The API will also give you access to the underlying object creation to do with those what you please. We hope these options will let you integrate the package into your pre-existing bash and Python scripts, and give you some handy tools to use in your day-to-day work with running NWP systems.
+
+Configuration Management
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+| **CLI**: ``uw config -h``
+| **API**: ``import uwtools.api.config``
+
+The uw config tool suite helps you compare, transform, modify, and even validate your configuration files. The package supports YAML, shell, Fortran namelist, and INI file formats. Each of these formats allows you to specify values as Jinja2 templates to enable referencing other values for direct substitution or derived information (math, path building, string manipulation, etc.).
+
+Compare Mode
+............
+
+When the Linux diff tool just doesn't work for comparing unordered namelists with mixed-case keys, this is your go-to! It also works on the other file types, but the Fortran namelists are the *real* catalyst behind this gem!
+
+| See CLI examples here: :any:`Examples<compare_configs_cli_examples>`
+| This mode is not supported in the API.
+
+
+Realize Mode
+............
+
+This mode does the work of rendering values created by Jinja2 templates, and lets you override settings in one file with those from others, not necessarily with the same file type. With ``uwtools``, you can even reference the content of other files to build up a configuration file from its pieces.
+
+| See CLI examples here: :any:`Examples<realize_configs_cli_examples>`
```

Comment:
I wasn't sure what to do here about the API docs. I'd love to hear your ideas. Will we have examples or just the autodocs? Should I link to that content?

Reply:
We're in the process of adding examples, but I'm not sure we want to reiterate this much of the content here?